frontend/src/app/components/UpdateList/index.js

```diff
@@ -138,7 +138,7 @@ export default class UpdateList extends React.Component {
     const shouldShowMoreNewsButton = !showMoreNews && hasStaleNewsUpdates;
 
     const shownNewsUpdates = shouldShowStaleNewsUpdates
-      ? [].concat(freshNewsUpdates, staleNewsUpdates.slice(0, 5))
+      ? [].concat(freshNewsUpdates, staleNewsUpdates.slice(0, 10))
```

Comment:
This was already a bit of hack, but since we're now using a page to display news, I went ahead and puffed this number up a bit.